solr/core/src/test/org/apache/solr/response/TestRetrieveFieldsOptimizer.java

```diff
@@ -616,7 +615,7 @@ List<String> getValsForField() {
         break;
 
       default:
-        fail("Found no case for field " + name + " type " + type);
+        SolrTestCaseJ4.fail("Found no case for field " + name + " type " + type);
```

Comment:
because this is in `RetrieveField` and doesn't extend anything with that method. This uses `SolrTestCaseJ4` since it is the correct class in the chain that was already imported. `SolrTestCaseJ4` ends up inheriting from Assert anyway.